### PR TITLE
@pepopowitz => [AutoSuggest] Rough implementation of artist handling for previews

### PR DESCRIPTION
**Pull request by mzikherman**
Here's an example of how we might start to organize the components for the RHS 'additional context' stuff for autosuggest.

Here's the folder structure: https://github.com/mzikherman/reaction-force/tree/search_context/src/Components/SearchAutoSuggest

This is totally unstyled for now! 😅

### CODE REVIEW

File: src/Components/SearchAutoSuggest/AdditionalContext/Grids/Artist.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import { ArtistSearchAdditionalContext_artist } from "__generated__/ArtistSearchAdditionalContext_artist.graphql"
```
Comment (mzikherman) [inline src/Components/SearchAutoSuggest/AdditionalContext/Grids/Artist.tsx]: In this folder, we can create all the components we need for the different 'contexts' for the RHS. So, one person can be working on the artist case (show collections OR artworks), one person can be working on the partner case (show rep artists), one person can be working on the sale case, and so on. If we split up those cases, we'll all be working on separate files and can parallelize some of that work among 4 of us (vs just 2).

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 5b0c7d43e0eb61f11f64cf96a294f32da58f2d87

https://deploy-preview-1989--artsy-reaction.netlify.com

**Comment by mzikherman** [comment]
Here's how the stories are currently:

screen shot 2019-02-05 at 12 04 02 pm

**Comment by mzikherman** [comment]
Ah, check out the PR storybooks: https://deploy-preview-1989--artsy-reaction.netlify.com/?selectedKind=Components%2FSearchAutoSuggest%2FContext%2FSale&selectedStory=A%20sale&full=0&addons=0&stories=1&panelRight=0 for an idea.

**Comment by mzikherman** [comment]
Ok @pepopowitz @jonallured merging this as discussed!

**Pull request merged**